### ISSUE TITLE
unzip sucessfully in 1.x but failed in latest version

### ISSUE THREAD

**Issue by jacktang** [srikanth-lingala/zip4j#65]
Hello,

I created the zip using c# `ICSharpCode.SharpZipLib.Zip` with password. And it could be unzipped when I extracted it using zip4j `version 1.3.2`. I upgrade it 2.x in order to support stream. And some exception when I extract the same zip file:

Code:
```java
    try {
      ZipFile zipFile = new ZipFile(source);
      if (zipFile.isEncrypted()) {
        zipFile.setPassword(password);
      }
      zipFile.extractAll(destination);
    } catch (ZipException e) {
      e.printStackTrace();
    }
```

Exception:
```java
net.lingala.zip4j.exception.ZipException: java.io.IOException: Negative seek offset
	at net.lingala.zip4j.tasks.AsyncZipTask.performTaskWithErrorHandling(AsyncZipTask.java:49)
	at net.lingala.zip4j.tasks.AsyncZipTask.execute(AsyncZipTask.java:36)
	at net.lingala.zip4j.ZipFile.extractAll(ZipFile.java:431)
	at app.util.ZipMaker.unzip(ZipMaker.java:108)
```

The sample zip [hello.zip](https://github.com/srikanth-lingala/zip4j/files/3616840/hello.zip) with password `Shu1an@2019GTS`

**Comment by jacktang** [comment]
@srikanth-lingala if the bug was fixed, would you please bump new version?

**Comment by srikanth-lingala** [comment]
@jacktang I will make a new release over the weekend. There is just one more feature that I want to include in the next release, and also an issue that is reported a day or two back. I want to have a look at them and make a release.

**Comment by srikanth-lingala** [comment]
Fixed in v2.2.1
